vendor/github.com/xanzy/go-cloudstack/cloudstack/AffinityGroupService.go

```diff
@@ -98,7 +98,7 @@ func (p *CreateAffinityGroupParams) SetType(v string) {
 	if p.p == nil {
 		p.p = make(map[string]interface{})
 	}
-	p.p["affinityGroupType"] = v
+	p.p["type"] = v
```

Comment:
Hmm, this doesn't look good...

You should never change a vendored package directly. This would mean that we now effectively forked this package, without it being documented anywhere.

If this is somethings that needs to be fixed in the `go-cloudstack` package, it should be fixed in there (at `github.com/xanzy/go-cloudstack`). And then after that we can update the vendored package in here to pull in the fix.